core/templates/dev/head/pages/story-viewer-page/story-viewer-page.directive.ts

```diff
@@ -184,7 +181,31 @@ angular.module('oppia').directive('storyViewerPage', [
               if (!storyNodes[i].isCompleted() &&
                 (storyNodes[i].getId() !==
                 ctrl.storyPlaythroughObject.getNextPendingNodeId())) {
-                thumbnailColor = 'grey';
+                if (
+                  storyNodes[i].getExplorationSummaryObject(
```

Comment:
Bit clumsy to put all this stuff in the "if" -- maybe compute it in a separate variable and then condition on that variable.

Reply:
Removed this condition.